<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpPageClass</class>
 <widget class="QMainWindow" name="SignUpPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignUpPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/GymPadelSystem/Resources/Gym.png&quot;);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="SUWidget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>320</width>
      <height>610</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
background-color: rgba(0,0,0,0.7);
border-radius: 25px;</string>
    </property>
    <widget class="QLineEdit" name="FNTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color: rgba(250,250,250,0.7);
padding-left: 15px;
font-size: 15px;
color: black;</string>
     </property>
     <property name="placeholderText">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MNTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color: rgba(250,250,250,0.7);
padding-left: 15px;
font-size: 15px;
color: black;</string>
     </property>
     <property name="placeholderText">
      <string>Middle Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LNTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color: rgba(250,250,250,0.7);
padding-left: 15px;
font-size: 15px;
color: black;</string>
     </property>
     <property name="placeholderText">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>121</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color: rgba(250,250,250,0.7);
padding-left: 15px;
font-size: 15px;
color: black;</string>
     </property>
     <property name="placeholderText">
      <string>Day</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SignUpButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>530</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background: none;
background-color: rgba(135, 206, 235,0.9);
color: black;
border-radius: 25px;
}
QPushButton:hover
{
border: 3px solid rgba(135, 206, 235,0.9);
background-color: rgba(250, 250, 250,0.8);
}</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>320</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
font-size: 30px;
color: rgba(135, 206, 235,0.8)</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="MTextBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>121</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color: rgba(250,250,250,0.7);
padding-left: 15px;
font-size: 15px;
color: black;</string>
     </property>
     <property name="placeholderText">
      <string>Month</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="YTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color: rgba(250,250,250,0.7);
padding-left: 15px;
font-size: 15px;
color: black;</string>
     </property>
     <property name="placeholderText">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
font-size: 20px;
color: rgba(135, 206, 235,0.8)</string>
     </property>
     <property name="text">
      <string>Date Of Birth</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMLabel_E">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>520</y>
       <width>221</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
font-size: 15px;
background-color: rgba(250, 0, 0, 0.5);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ExiButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>800</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background: none;
background-color: rgba(255, 51, 51,0.9);
color: white;
border-radius: 25px;
}
QPushButton:hover
{
border: 2px solid rgba(255, 51, 51,0.9);
background-color: rgba(250, 250, 250,0.8);
color: rgb(255, 51, 51);
}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>FNTextBox</tabstop>
  <tabstop>MNTextBox</tabstop>
  <tabstop>LNTextBox</tabstop>
  <tabstop>DTextBox</tabstop>
  <tabstop>MTextBox</tabstop>
  <tabstop>YTextBox</tabstop>
  <tabstop>SignUpButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
